### PR TITLE
Disable tests if the k8s version is not supported

### PR DESCRIPTION
**Pull request by hellozee**

### CODE REVIEW

File: tetrateci/test_1.8.sh [hunk 1]
```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+export MINOR_VER=$REL_BRANCH_VER
```
Comment (deva26) [inline tetrateci/test_1.8.sh:2]: We don't export the vars in the create and delete cluster scripts. May be you can add all the env to github action and bring the create/delete cluster scripts and test script on parity and standardized in using the env vars.

Comment (hellozee) [reply tetrateci/test_1.8.sh:2]: not possible to add the enviroment variables for the whole workflow since the variables depend on the output of the jobs

Comment (deva26) [reply tetrateci/test_1.8.sh:2]: Got it. Add the vars to the individual jobs or steps and make sure all of them in parity. Think of it from this angle - create_eks_cluster.sh calls this version script, but does not have his variables defined and exported inside of them.

**Review by deva26 — APPROVED** [review]
Thanks.

**Pull request merged**